### DISCUSSION TITLE
✨ Update: German and Portuguese UI

### DISCUSSION THREAD

**Discussion by aikoii** [GitbookIO/community#649] [Announcements]
Hey all! 👋

We're excited to let you know that you can now localize the UI in German and Portuguese 🎉

A big shoutout to @burghoffdavid for helping with the German 🇩🇪 and to @rodrcastro for the Brazilian Portuguese translation 🇧🇷💙!

**Comment by gutwrinch** [comment]
I'm just excited to have personal interaction (ACTION)
